packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/AccessibilityGuide.6.doc.mdx

```diff
@@ -0,0 +1,203 @@
+
+## Accessibility Considerations
+
+Terra Clinical Result has several variants designed to either display specific results or for use within a flowsheet. The table below lists each variant and its use:
+
+| Variant | Use |
+| --- | --- |
+| Clinical Result | The default Clinical Result component intended for displaying a single result value |
+| Clinical Result Blood Pressure | Intended for displaying Blood Pressure Vital Sign results |
+| Flowsheet Result Cell | Specifically for the grid display of a result collection |
+| Result Name Header Cell | Standardized view for the Result Name and Unit of Measure in the row header for each row within a flowsheet |
+| Result Time Header Cell | Standardized view for the Date/Time Header cells for each bucketed time column within a flowsheet |
+
+The following sections go into more detail on accessibility considerations for each variant.
+
+----
+
+### Code Considerations
+
+#### Clinical Result Blood Pressure with Additional Displays
+
+Clinical Result Blood Pressure creates a standardized view for single result values added to a patient's medical chart. Along with the blood pressure results, we have the option to display additional details like concept names (Systolic, Diastolic) and the date and time of the results. Visually we can see that this information is related but it is possible that screen reader users may not understand the whole cell as a group.
+
+<Notice variant="important" ariaLevel="5">
+
+It is highly recommended to set the optional `isBloodPressureGrouped` [prop](https://engineering.cerner.com/terra-ui/components/terra-clinical-result/clinical-result/clinical-result-blood-pressure#clinical-result-blood-pressure-props) to true when additional information is displayed with the blood pressure results. This will semantically relate the blood pressure results and additional displays as a group context.
+
+**Code Example:**
+
+<div aria-label="Example demo">
+  <ClinicalResultBloodPressureExtraDisplays />
+</div>
+<div aria-label="Example code">
+
+    import React from 'react';
+    import { ClinicalResultBloodPressure } from 'terra-clinical-result/lib';
+
+    const bloodpressureResultValue = {
+      id: '111',
+      systolic: {
+        eventId: '111.1',
+        result: {
+          value: '120',
+          unit: 'mmHg',
+        },
+        conceptDisplay: 'Systolic',
+        datetimeDisplay: 'Nov 23, 2019 13:31:31',
+      },
+      diastolic: {
+        eventId: '111.2',
+        result: {
+          value: '80',
+          unit: 'mmHg',
+        },
+        conceptDisplay: 'Diastolic',
+        datetimeDisplay: 'Nov 23, 2019 13:31:31',
+      },
+      isBloodPressureGrouped: true,
+    };
+
+    export default () => <ClinicalResultBloodPressure {...bloodpressureResultValue} />;
+
+</div>
+
+This example shows how to set up the optional `isBloodPressureGrouped` prop in order to group the Blood Pressure results with additional displays.
+
+</Notice>
```

Comment:
This seems like a lot of information for notice banner. I think we should reduce it to a short summary/statement. The code example should be part of the main document.

CC: @eawww thoughts?

Reply:
@sdadn I agree. The first paragraph is good content for the 'important' notice. It could even say "See example below." at the end. The "important" notice is meant to draw a reader's eye to particularly important information. If half the page is 'important' then the real important bits get lost.
This would also come with the collapsible, syntax highlighted code example that the normal example widget offers.

This would apply to both examples on the page.

I'd also add that the "This example shows..." lines that come after the examples would probably be better before the examples.

Reply:
@eawww - just to clarify:
> This would also come with the collapsible, syntax highlighted code that the normal example widget offers.

Is there a component like `<Notice>` that I should be using for this? I moved the code example outside of the Notice block but it currently looks the same as it did inside. I tried using ```jsx like I saw in [other accessibility guides](https://github.com/cerner/terra-clinical/blob/main/packages/terra-clinical-detail-view/src/terra-dev-site/doc/clinical-detail-view/AccessibilityGuide.2.doc.mdx#L46) which does add syntax highlighting but doesn't appear to be collapsible.

Reply:
@zxeleven Sorry the usual examples I was thinking of import from JSX files with `?dev-site-example` appended to the end of the path. For this purpose, either way would be fine so long as it's outside of the Notice.

Reply:
I went ahead and used `?dev-site-example` for these examples as well: https://github.com/cerner/terra-clinical/pull/884/commits/4f06978a1a8827cbf3dceee49ba6c8df54b3a912